eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantEntity.java

```diff
@@ -66,6 +66,10 @@
     protected SubmittedVariantEntity() {
     }
 
+    public SubmittedVariantEntity(Long accession, String hashedMessage, ISubmittedVariant model) {
```

Comment:
do we need this constructor?

Reply:
it seems we don't need it. Deleting it...